libbeat/common/cfgwarn/removed.go

```diff
@@ -64,3 +63,23 @@ func CheckRemoved5xSetting(cfg *common.Config, setting string) error {
 
 	return fmt.Errorf("setting '%v' has been removed", current.PathOf(name))
 }
+
+// CheckRemoved5xSettings prints a warning if the obsolete setting is used.
+func CheckRemoved5xSettings(cfg *common.Config, settings ...string) error {
```

Comment:
Should we remove now in master all 5.x warnings? I don't expect people to upgrade from 5.x to 7.x directly.

Reply:
Yes I wanted to do it in another PR